### DISCUSSION TITLE
400: Invalid label name - invalidArgument

### DISCUSSION THREAD

**Discussion by mtiraferri** [GAM-team/got-your-back#461] [Q&A]
Tryng to restore 22Gb mbox, suddenly i have this error:
Restoring from 22.69gb file c:\tmp\tb\recupero.mbox...
Creating label Da leggere
400: Invalid label name - invalidArgument

Google workspace user,
windows 11 installation.
Thanks for any help.
Marco

**Comment by mdesdin** [comment]
have you found a solution?

**Comment by mdesdin** [comment]
I am facing the same error:
Creating label No leídos                                                        
400: Invalid label name - invalidArgument

This is a mbox created by Takeout.
No leídos means Unread, it is not really an existing label.

I have tried playing with all possible variants of writing "No leídos" with --extra-reserved-labels and --extra-system-labels, to no avail…

**Comment by mdesdin** [comment]
Takeout translates labels, GMAIL native/system labels, and then GMAIL is not aware when importing back. Or maybe there is a way to set the language in GYB when importing?

**Comment by mdesdin** [comment]
For spanish, these labels need to be replaced:
Abierto/Opened
Archivado/Archived
Borradores/Drafts
Categor=C3=ADa_Compras/Category Purchases
Categor=C3=ADa_Viajes/Category Travel
Categor=C3=ADa:_foros/Category Forums
Categor=C3=ADa:_novedades/Category Updates
Categor=C3=ADa:_personal/Category Personal
Categor=C3=ADa:_promociones/Category Promotions
Categor=C3=ADa:_social/Category Social
Chat/Chat
Confirmaci=C3=B3n_de_lectura_enviada/Read Receipt Sent
Destacados/Starred
Enviados/Sent
Env=C3=ADo_programado/Scheduled Send
Importante/Important
No_le=C3=ADdos/Unread
Notes/Notes
Papelera/Trash
Plantilla de respuesta/Template Response
Pospuesto/Snoozed
Recibidos/Inbox
Silenciado/Muted
Spam/Spam

(no the ones that are the same, like Notes/Notes)

something like this should do it: sed -i -e '/X-Gmail-Labels/s/No_le=C3=ADdos/Unread/' mbox-file-here
